### ISSUE TITLE
Aliased numeric systems are not working properly

### ISSUE THREAD

**Issue by cquiroz** [cquiroz/scalajs-locales#20]
Some examples are the locales

bn
fa
os
uz

**Comment by cquiroz** [comment]
The reason for this problem is that the JVM defaults to the latn unless explicitly requested, e.g. bn defaults to the "beng" numeric system, however the call

```
val bn = Locale.forLanguageTag("bn")
DecimalFormatSymbols.getInstance(bn).getNaN()
res13: String = NaN
```

While the result with pure cldr should be
```
val bn2 = Locale.forLanguageTag("bn-u-nu-beng")
DecimalFormatSymbols.getInstance(bn2).getNaN()
res12: String = সংখ্যা না
```

**Comment by cquiroz** [comment]
Fixed by #26
